intel_extension_for_transformers/neural_chat/docs/notebooks/langchain_extension_api.ipynb

```diff
@@ -79,32 +86,87 @@
     "from langchain_core.vectorstores import VectorStoreRetriever\n",
-    "from langchain_core.documents import Document\n",
     "from langchain_community.embeddings import HuggingFaceBgeEmbeddings\n",
     "from transformers import AutoModelForCausalLM, AutoTokenizer, pipeline\n",
+    "from langchain.vectorstores import Chroma\n",
+    "import time\n",
+    "\n",
+    "loader = PyPDFLoader(\"./Intel_AR_WR.pdf\")\n",
+    "langchain_documents = loader.load_and_split()\n",
+    "start = time.time()\n",
+    "embeddings = HuggingFaceBgeEmbeddings(model_name=\"BAAI/bge-base-en-v1.5\")\n",
+    "knowledge_base = Chroma.from_documents(documents=langchain_documents, embedding=embeddings, persist_directory='./out')\n",
+    "tokenizer = AutoTokenizer.from_pretrained(\"meta-llama/Llama-2-7b-chat-hf\")\n",
+    "model = AutoModelForCausalLM.from_pretrained(\"meta-llama/Llama-2-7b-chat-hf\")\n",
+    "pipe = HuggingFacePipeline(pipeline=pipeline(\"text-generation\", model=model, tokenizer=tokenizer, max_new_tokens=128))\n",
+    "retriever = VectorStoreRetriever(vectorstore=knowledge_base, search_type='mmr', search_kwargs={'k':1, 'fetch_k':5})\n",
+    "retrievalQA = RetrievalQA.from_llm(llm=pipe, retriever=retriever)\n",
+    "result = retrievalQA({\"query\": \"What is IDM 2.0?\"})\n",
+    "end = time.time()\n",
+    "print(f' --- retrieval QA took {end - start:.3f} sec')\n",
+    "print(result)"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "Chatbot code with ITREX langchain extension APIs:"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "from langchain_community.llms.huggingface_pipeline import HuggingFacePipeline\n",
+    "from langchain.chains import RetrievalQA\n",
+    "from langchain_core.vectorstores import VectorStoreRetriever\n",
+    "from langchain_core.documents import Document\n",
+    "from intel_extension_for_transformers.langchain.embeddings import HuggingFaceBgeEmbeddings\n",
+    "from transformers import AutoModelForCausalLM, AutoTokenizer, pipeline\n",
     "from intel_extension_for_transformers.langchain.vectorstores import Chroma\n",
     "from intel_extension_for_transformers.neural_chat.pipeline.plugins.retrieval.parser.parser import DocumentParser\n",
+    "import time\n",
     "\n",
     "document_parser = DocumentParser()\n",
-    "input_path=\"./412439%281%29_12_Intel_AR_WR.pdf\"\n",
+    "input_path=\"./Intel_AR_WR.pdf\"\n",
     "data_collection=document_parser.load(input=input_path)\n",
     "documents = []\n",
     "for data, meta in data_collection:\n",
     "    doc = Document(page_content=data, metadata={\"source\":meta})\n",
     "    documents.append(doc)\n",
-    "embeddings = HuggingFaceBgeEmbeddings(model_name=\"BAAI/bge-base-en-v1.5\")\n",
+    "start = time.time()\n",
+    "# Intel/bge-base-en-v1.5-sts-int8-static is private now, so we need to load it from local.\n",
```

Comment:
Intel/bge-base-en-v1.5-sts-int8-static is private now, so we need to load it from local.
=> 
load Intel/bge-base-en-v1.5-sts-int8-static from local